src/control/control.cpp

```diff
@@ -154,6 +169,16 @@ QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getControl(
     return nullptr;
 }
 
+//static
+QSharedPointer<ControlDoublePrivate> ControlDoublePrivate::getDefaultControl() {
+    auto defaultCO = QSharedPointer<ControlDoublePrivate>(s_defaultCO);
+    if (!defaultCO) {
+        defaultCO = QSharedPointer<ControlDoublePrivate>(new ControlDoublePrivate());
+        s_defaultCO = defaultCO;
```

Comment:
What happens if two or more default COs are created due to race conditions?

Reply:
Than we have two or more default created COs. Not a problem.
But I can think about a way to avoid it.

Reply:
Just add a comment where the race condition might appear to document that the code been written with the awareness of it.